client/public/index.html

```diff
@@ -13,6 +14,37 @@
 
       gtag('config', 'UA-146085874-1');
     </script>
+    <!-- LeadOut-specific Pendo -->
+    <script>
+      (function (apiKey) {
+        (function (p, e, n, d, o) {
+          var v, w, x, y, z;
+          o = p[d] = p[d] || {};
```

Comment:
can you add a comment to this block? looks strange with all of the single letter variables and very unclear whats going on